app/services/facebook_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FacebookService
+  def initialize(access_token)
+    @access_token = access_token
+  end
+
+  def profile
+    client.get_object('me?fields=email,first_name,last_name')
+  end
+
+  def client
+    Koala::Facebook::API.new(@access_token, ENV['FACEBOOK_SECRET'])
```

Comment:
We prefer to use `ENV.fetch('KEY')` instead of `ENV['KEY']`

The difference is that `ENV['KEY']` returns nil if the env variable does not exist and `ENV.fetch('KEY')` raises an exception in that case